src/frame/mod.rs

```diff
@@ -20,6 +20,24 @@ pub use plane::*;
 
 const FRAME_MARGIN: usize = 16 + SUBPEL_FILTER_SIZE;
 
+/// Override the frame type decision
+///
+/// Only certain frame types can be selected.
+#[derive(Debug, PartialEq, Clone, Copy)]
+pub enum FrameTypeOverride {
+  /// Do not force any decision.
+  No,
```

Comment:
Is there a point in having `No` instead of making `frame_type_override` an Option?

Reply:
`Some(FrameTypeOverride::Key)` is fairly verbose.

Reply:
Seems like the idiomatic way to do it though.

Reply:
One issue is that I think overriding to a Key or S-frame should be allowed (or maybe even intra only), but overriding to a inter frame shouldn't. So that doesn't fit the Option idiom very well.